2024-RIKEN-AWS/JupyterNotebook/docker/Dockerfile.spawn

```diff
@@ -1,4 +1,4 @@
-FROM fluxrm/flux-sched:jammy
+FROM fluxrm/flux-sched:focal
```

Comment:
Why did you change to focal? Jammy is newer (and it's going to last longer so I tend to use it).

Reply:
Because of requirements for some dependencies of DLIO (namely Tensorflow), we need certain packages that are not available on Jammy. @hariharan-devarajan could give you a better sense of what those packages are than I could.

Reply:
okay that's reasonable - focal goes EOL in about a year, so I'm OK with one more year of tutorial! We will likely need to update it next year.